tests/unit/conftest.py

```diff
@@ -374,12 +375,44 @@ def full_record():
         },
         thesis={
             'university': 'I guess important',
-        }
+        },
+        files=[],
     )
     record['$schema'] = 'http://zenodo.org/schemas/records/record-v1.0.0.json'
     return record
 
 
+@pytest.fixture()
```

Comment:
see https://github.com/inveniosoftware/invenio-records-files/blob/master/tests/conftest.py#L94-L119 and https://github.com/inveniosoftware/invenio-records-files/blob/master/tests/test_invenio_records_files.py#L189-L190